### ISSUE TITLE
define API for html.out OOB

### ISSUE THREAD

**Issue by s-u** [att/rcloud#1319]
Now that we have a nice "html.out" OOB R code can actually generate HTML directly. Previously this was only possible with Markdown. We should add a function to wrap this ... ideas for names? `rcloud.html`?

**Comment by gordonwoodhull** [comment]
I guess `rcloud.output.html` is too wordy.

**Comment by gordonwoodhull** [comment]
`rcloud.html.out` to match our old `rcloud.out`

**Comment by s-u** [comment]
Sure, I'm lazy so I like short names but `rcloud.html.out` is more specific and thus hopefully clearer.

**Comment by s-u** [comment]
Finalized and documented
